lib/src/main/java/com/couchbase/lite/internal/CouchbaseLiteInternal.java

```diff
@@ -74,6 +74,8 @@ public static void init(@NonNull MValue.Delegate mValueDelegate, @Nullable Strin
 
         NativeLibrary.load();
 
+        C4Base.debug();
```

Comment:
curious, what is this? is there a c4-api for debugging, i haven't seen a function like this in iOS.

Reply:
Yeah... brand new.  It is intended to make Core produce a stack-trace, when it fails.  I'm not sure it is working....